atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
is the orchestrate_build step necessary? the help_file is just stored as annotations['help_file'] and gets passed on to koji_import without any specific handling by orchestrate_build.